api/openapi-spec/swagger.json

```diff
@@ -10200,11 +10200,7 @@
           "type": "string"
         },
         "externalTrafficPolicy": {
-          "description": "externalTrafficPolicy denotes if this Service desires to route external traffic to node-local or cluster-wide endpoints. \"Local\" preserves the client source IP and avoids a second hop for LoadBalancer and Nodeport type services, but risks potentially imbalanced traffic spreading. \"Cluster\" obscures the client source IP and may cause a second hop to another node, but should have good overall load-spreading.\n\nPossible enum values:\n - `\"Cluster\"` specifies node-global (legacy) behavior.\n - `\"Local\"` specifies node-local endpoints behavior.",
-          "enum": [
-            "Cluster",
-            "Local"
-          ],
```

Comment:
same question here... is the externally visible change correct?

Reply:
No. However, the problem was the constant aliases breaking enum OpenAPI generator. I am now working on a fix.

Reply:
The problem is with `k8s.io/gengo`, trying to fix.